app/helpers/grade_entry_forms_helper.rb

```diff
@@ -2,26 +2,6 @@
 
 module GradeEntryFormsHelper
 
-  # Release/unrelease the marks for a grade entry form for the given
-  # grade_entry_students
-  # Return the number of GradeEntryStudents that have been updated successfully
-  def set_release_on_grade_entry_students(grade_entry_students, release, errors)
-    numGradeEntryStudentsChanged = 0
-    grade_entry_students.each do |grade_entry_student|
-      begin
-        grade_entry_student.released_to_student = release
-        unless grade_entry_student.save
-          raise I18n.t('grade_entry_forms.grades.update_error',
```

Comment:
@mishaschwartz can you remove this key as well? (This is causing the failing test.)